nginx/config-files/hydroshare-local-nginx.conf.template

```diff
@@ -108,4 +108,9 @@ server {
     location @maintenance {
         rewrite ^(.*)$ /maintenance_on.html break;
     }
+
+    error_page 500 502 504 @server_error;
+    location @server_error {
+        rewrite ^(.*)$ /server_error_on.html break;
```

Comment:
I suppose this path is perhaps not correct given `root` directive. I'm not good with NGINX please LMK if this is bad

Reply:
The updated changes look correct to me.  It might be good to get @cbcunc to look at this when he is back from vacation before merging.